### ISSUE TITLE
module "compendium-folders" declares media.link which is unsupported, media.url should be used

### ISSUE THREAD

**Issue by keldrilh** [earlSt1/vtt-compendium-folders#220]
Good morning,

I put in the subject line the error line provided to me by Foundry (v11.313)
Thank you and have a nice day

**Comment by thephule** [comment]
Same Issue

**Comment by Nergaahl** [comment]
I have the same issue, running the latest stable version, but peeking into the module.json I see this:
"media": [
    {
      "type": "cover",
      "url": "https://raw.githubusercontent.com/earlSt1/vtt-compendium-folders/master/example.png",
      "loop": false,
      "flags": {}
    }

and I couldn't find any other instance of media in any other files.

**Comment by NikitosAseev** [comment]
> I have the same issue, running the latest stable version, but peeking into the module.json I see this: "media": [ { "type": "cover", "url": "https://raw.githubusercontent.com/earlSt1/vtt-compendium-folders/master/example.png", "loop": false, "flags": {} }
> 
> and I couldn't find any other instance of media in any other files.

Indeed, the problem was with the example.png file. I simply converted it into a webp format and uploaded it to Discord to insert the link, and ultimately, the warning disappeared.

https://cdn.discordapp.com/attachments/852608241270521897/1180282801170423818/example.webp?ex=657cdb04&is=656a6604&hm=c7dcc5ccf8ed7b6df49897d2602f99d6542d3d115cb4c22db57e4111aa095336&.webp
